### PR TITLE
update to 0.4.6

### PR DESCRIPTION
**Pull request by boldorider4**
* [x] checked license
* [x] updated dependencies with [upstream](https://github.com/pythological/python-cons/blob/main/requirements.txt)
* [x] updated about section

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,37 +1,46 @@
...
   host:
     - pip
-    - python >=3.6
+    - python
     - setuptools
+    - wheel
   run:
-    - python >=3.6
+    - python
     - logical-unification >=0.4.1
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml]: should be >= 0.4.0 ? https://github.com/pythological/python-cons/blob/bafb13f9f25da3b13deb2b58d9d0709a259ef697/setup.py#L12C5-L12C53

**Review by skupr-anaconda — APPROVED** [review]
LGTM but `logical-unification` pinning

File: recipe/meta.yaml (hunk 1 above)
Comment (boldorider4) [reply recipe/meta.yaml]: good catch!

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Pull request merged**